### DISCUSSION TITLE
Rush build through Github Action

### DISCUSSION THREAD

**Discussion by zainulhassan815** [shreyashsaitwal/rush-cli#36] [Ideas]
Hey @shreyashsaitwal hope you are doing great. I had this in mind for quite a long time. Do you have any plans on creating a github action that would build the extension. This would be really helpful for testing builds etc.

**Comment by shreyashsaitwal** [comment]
I don't think we need to create an Action for this. You can already do that by installing Rush during your workflow run and running the build command as you'd normally do locally.
